neonvm/runner/main.go

```diff
@@ -558,6 +583,22 @@ func runInitScript(logger *zap.Logger, script string) error {
 	return nil
 }
 
+func qemuDiskArgs(id string, file string, extraOptions string, deviceExtraOptions string, diskIdxCnt *int) []string {
```

Comment:
Nit: having an int pointer and mutating it doesn't look good. Also, increasing the `diskIdxCnt` seems out of scope for this function. To avoid this you can define a closure in main where it calls `qemuDiskArgs` and then increases the counter, which is locally defined there.

Reply:
sounds good :)